src/components/tabs/tabs.njk

```diff
@@ -1,35 +1,35 @@
-<div class="rvt-tabs" data-tabset="tabset-1">
+<div class="rvt-tabs" data-rvt-tabs="tabset-1">
     <div class="rvt-tabs__tablist" role="tablist" aria-label="Rivet tabs">
         <button class="rvt-tabs__tab"
```

Comment:
As a separate refactoring pass, I'd like to take a look and see if we can do a bit more to focus on taking some of the set-up work off of the developers writing the markup for these components.

For example adding all the roles, states and attributes related to accessibility and interactivity. Could we do that setup when the element is connected to the DOM? That way the markup the developer needs to grab is much simpler and less error prone when setting up in their own templates.

These tab buttons/panels are a good example. we could probably just add/set up all the aria roles and states in our component code instead of the developer having to make sure they're there and matched up correctly, etc.